frontend/src/pages/Logout.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import {Redirect} from "react-router-dom";
+import Cookies from 'js-cookie';
+import * as LoginActions from "../actions/LoginActions";
+import {connect} from "react-redux";
+
+class Logout extends Component {
+
+    constructor(props) {
+		super(props);
+		Cookies.remove("username");
```

Comment:
Can you reindent lines 10 and 11 to be aligned with the other lines? Just to keep things standardized.

Reply:
i think this is just git being crazy